src/repos/ArticleRepo.ts

```diff
@@ -48,30 +49,52 @@ const getArticlesAfterDate = async (since: string, limit = DEFAULT_LIMIT): Promi
   );
 };
 
-/** A function to compare the trendiness of articles.
- *
- * Trendiness is computed by taking the number of total shoutouts an article
- * has received and dividing it by the number of days since its been published.
- *
- * Sorts in order of most trendy to least trendy.
- *
- */
-export const compareTrendiness = (a1: Article, a2: Article) => {
-  const presentDate = new Date().getTime();
-  const a1Score = a1 != null ? a1.shoutouts / (presentDate - a1.date.getTime()) : 0;
-  const a2Score = a2 != null ? a2.shoutouts / (presentDate - a2.date.getTime()) : 0;
-  return a2Score - a1Score;
-};
-
 /**
  * Computes and returns the trending articles in the database.
  *
  * @function
  * @param {number} limit - number of articles to retrieve.
  */
 const getTrendingArticles = async (limit = DEFAULT_LIMIT): Promise<Article[]> => {
-  const articles = await ArticleModel.find({}).exec();
-  return articles.sort(compareTrendiness).slice(0, limit);
+  const articles = await ArticleModel.find({ isTrending: true }).exec();
+  return articles.slice(0, limit);
+};
+
+/**
+ * Refreshes trending articles.
+ */
+const refreshTrendingArticles = async (): Promise<Article[]> => {
+  // Set previous trending articles to not trending
+  const oldTrendingArticles = await ArticleModel.find({ isTrending: true }).exec();
+  oldTrendingArticles.forEach(async (a) => {
+    try {
+      const article = await ArticleModel.findById(new ObjectId(a._id)); //eslint-disable-line
+      article.isTrending = false;
+      await article.save();
+    } catch (err) {
```

Comment:
dry `try` `catch`es are not recommended -- you should at least for one level catch for an expected error, or you should only just be checking if you can find the article with the id in the first place